src/service.js

```diff
@@ -1,20 +1,39 @@
 let express = require('express');
 let http = require('http');
+let timeout = require('connect-timeout');
 
 let server;
 
 exports.start = function start(port = 8080) {
   let app = express();
+
+  // We shouldn't timeout log requests.
+  app.get('/log', require('./log'));
```

Comment:
When writing to `/log` lands, moving this line won't be helpful because `GET` shouldn't timeout, but the others should. I think `connect-timeout` allows us to override per request, so depending on timing of PRs we may have to modify some things.